moovie_app/lists/tests.py

```diff
@@ -0,0 +1,63 @@
+from django.test import TestCase
+from django.urls import reverse
+from lists.models import Genre
+from lists.views import GenreViewSet
+from faker import Faker
+from rest_framework.test import APIRequestFactory, APITestCase
+from rest_framework.views import status
+
+# Create your tests here.
+
+fake = Faker()
+
+# Genre Tests
+class GenreTestCase(TestCase):
+  def test_genre(self):
+    self.assertEquals(Genre.objects.count(), 0)
+    Genre.objects.create(name='Action')
```

Comment:
Consider using https://pypi.org/project/model-bakery/ to generate test objects
example: baker.make(Genre, name='Action')

Reply:
https://model-bakery.readthedocs.io/en/latest/